lib/Server.js

```diff
@@ -77,6 +75,9 @@ Server.prototype.setup = function() {
   app.delete('/buckets/:bucket/token/:token', auth, this.routes.deleteBucketToken);
   app.get('/buckets/:bucket/token', auth, this.routes.listBucketTokens);
   app.get('/asset/:bucket/:assetName', auth, this.routes.serveFileAsset);
+  app.get('/asset-size/:bucket/:assetName', auth, this.routes.listAssetSize);
```

Comment:
This is kind of nit-picky, but I'd like to propose `/buckets/:bucket/asset/:assetName/size` just to make the URL more predictable.